lib/strava/web/api_response.rb

```diff
@@ -9,8 +9,20 @@ class ApiResponse
       # @param [Faraday::Response] http_response
       #
       def initialize(http_response)
-        @response = http_response
-        @ratelimit = Strava::Api::Ratelimit.new(@response)
+        @response = http_response.env.to_h
+        @ratelimit = Strava::Api::Ratelimit.new(http_response)
+      end
+
+      def to_h
+        { ratelimit: @ratelimit.to_h }
+      end
+
+      def to_json(options = {})
+        if ENV.fetch('STRAVA_JSON_INCLUDE_HTTP_RESPONSE', nil)
+          to_h.merge(response: @response).to_json(options)
+        else
+          to_h.to_json(options)
+        end
```

Comment:
I think it's very strange that `to_h` would behave differently from `to_json`. Why wouldn't we always include the response?